Core/Core/Discussions/DiscussionTopic.swift

```diff
@@ -88,10 +107,30 @@ public final class DiscussionTopic: NSManagedObject, WriteableModel {
             }
             return dict
         }
+        model.htmlURL = item.html_url
+        model.id = item.id.value
+        model.isAnnouncement = item.subscription_hold == "topic_is_announcement"
```

Comment:
What. I did not know this was a thing. I wonder if it's new. I seem to remember wishing for this when we added announcements and discussions to teacher the first time.

Reply:
This isn't new to this PR, I just reordered the properties.